### ISSUE TITLE
External Clock?

### ISSUE THREAD

**Issue by bsom** [tlgkccampbell/tinyvga#1]
I'm confused on how this works.

"The only external support circuitry which is required is a clock source with a frequency that depends on the desired resolution."

Where does this clock source come into play on an attiny85?

Do I simply supply a square wave to PB3 at the specified frequency of 6.29375 MHz?

**Comment by tlgkccampbell** [comment]
Yes, the MCU's system clock needs to be driven at the required frequency. On an ATtiny85, connect PB3 (CLKI) to the clock source and set your fuses to indicate that the device should use an external clock. Check out page 26 of the datasheet.

**Comment by bsom** [comment]
OK.  That's what I thought.

Now I just need to figure out how to generate the necessary clock!

Thanks!

**Comment by tlgkccampbell** [comment]
For my own projects I generally use a 25.175 MHz oscillator and divide it by 4:

https://www.mouser.com/c/passive-components/frequency-control-timing-devices/oscillators/standard-clock-oscillators/?frequency=25.175%20MHz&package%20%2F%20case=DIP-8&product%20type=Standard%20Oscillators

**Comment by bsom** [comment]
That's what I was thinking I'd do, too.  Just need to get my hands on some flip flops. :-)

**Comment by OddballNo3** [comment]
why not just use the internal clock?
